src/components/v5/frame/ColonyHome/partials/DashboardHeader/partials/ColonyLinks/ColonyLinks.tsx

```diff
@@ -84,10 +84,7 @@ const ColonyLinks = () => {
                 id: 'colony.tooltip.goToExternalLink',
               })}
             >
-              <ExternalLink
-                href={link}
-                className="text-gray-900 md:hover:text-blue-400"
-              >
+              <ExternalLink href={link} hasHover>
```

Comment:
md:hover:text-blue-400 - will be added by hasHover
text-gray-900 - will be taken by parent